### PR TITLE
Fixes #13903 - Move release version list to be host based

### PR DESCRIPTION
**Pull request by jlsherrill**

### CODE REVIEW

File: config/routes/overrides.rb [hunk 1]
```diff
@@ -79,6 +79,7 @@ def matches?(request)
               put :auto_attach
               get :product_content
               get :events
+              get :available_release_versions
```
Comment (waldenraines) [inline config/routes/overrides.rb]: Isn't there a corresponding systems route that should be removed?

Comment (jlsherrill) [inline config/routes/overrides.rb]: removed the releases one for systems, and some other lingerings ones

Comment (waldenraines) [inline config/routes/overrides.rb]: 👍

**Comment by waldenraines** [comment]
APJ

**Pull request merged**